docs/developer-docs/latest/plugins/email.md

```diff
@@ -100,8 +100,8 @@ yarn add @strapi/provider-email-sendgrid --save
 
 After installing your provider you will need to add some settings in `./config/plugins.js`. If this file doesn't exists, you'll need to create it. Check the README of each provider to know what configuration settings the provider needs.
 
-::: tip
-Make sure you have the correct spelling of the configuration filename, it is written in plural (with a trailing 's'): `plugins.js`.
+:::tip
+When using community providers, you need to pass the full package name to the `provider` key, only Strapi maintained providers can use the short-code eg: `provider: 'strapi-provider-email-mandrill'`
```

Comment:
```suggestion
When using community providers, pass the full package name to the `provider` key (e.g. `provider: 'strapi-provider-email-mandrill'`). Only Strapi-maintained providers can use the shortcode format (e.g. `provider: 'sendmail'`).
```

(to comply to [our style guide](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing))